pwa/src/pages/configurationsExportButton/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+import Layout from "../../components/common/layout";
+import ConfigurationsExportButton from "../../components/configurationsExportButton/ConfigurationsExportButton";
+
+const IndexPage = () => {
+  return (
+    <Layout title={"ConfigurationsExportButton"} subtext={"An overview of your endpoints objects"}>
+      <title>Gateway - ConfigurationsExportButton</title>
+      <main>
+        <div className="row">
+          <div className="col-12">
```

Comment:
Note: you wrapped the `ExportConfigurationsButton` component in an extra `<div>` (presumably because the button was too wide otherwise). You should fix that in this file, instead of inside the component.